srcpkgs/factor/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'factor'
 pkgname=factor
-version=0.97
+version=0.98
 revision=1
-only_for_archs="i686 x86_64 x86_64-musl"
+only_for_archs="i686 x86_64"
 build_style=gnu-makefile
 hostmakedepends="unzip pkg-config"
-makedepends="gtkglext-devel"
+makedepends="gtk+-devel gtkglext-devel"
+depends="gtk+-devel gtkglext-devel"
 nostrip_files="a.elf"
 short_desc="Concatenative programming language, similar to Forth"
 maintainer="B. Wilson <x@wilsonb.com>"
 license="BSD-2-Clause"
 homepage="http://factorcode.org/"
 distfiles="http://downloads.factorcode.org/releases/${version}/factor-src-${version}.zip"
-checksum=fd75e765590691c89b866e5265ae16bfb36d0e28eed095702ae1e206114663b8
+checksum=318fb8cdf84528fce17a83f7e9d76e55292ccc779c71d180d99a6465574d3706
 
 post_extract() {
 	mv "${XBPS_BUILDDIR}/factor" "${wrksrc}"
```

Comment:
No. Remove `post_extract()`, and simply use `wrksrc="factor"`

Reply:
Oh. Yeah. That's a relic from me wrangling with 0.97. Thanks!